_posts/2019-02-26-bazel-0.23.md

```diff
@@ -0,0 +1,211 @@
+    `--remote_timeout` would be interpreted as milliseconds instead of seconds.
+*   The incompatible flag
+    [`--incompatible_strict_action_env`](https://github.com/bazelbuild/bazel/issues/6648)
+    has been flipped to "false" again, as we discovered breakages for local
+    execution users. We'll need some more time to figure out the best way to
+    make this work for local and remote execution
+    ([#7026](https://github.com/bazelbuild/bazel/issues/7026)).
+*   The flag value `--test_sharding_strategy=experimental_heuristic` has been
+    removed.
+
+
+## Other changes
+
+*   Bazel binary is 20MB smaller by using a minimized embedded JDK.
+*   The embedded JDK version is upgraded to 11.0.2.
+*   Incompatible flag
+    [`--incompatible_auto_configure_host_platform`](https://github.com/bazelbuild/bazel/issues/7081)
+    has been added. When set, the default value of `--host_platform` and
+    `--platforms` will not be dependent on the configuration. This means that
+    setting `--cpu` or `--host_cpu` will not affect the target or host platform.
+*   Introduces
+    [`--local_ram_resources`](https://docs.bazel.build/versions/master/command-line-reference.html#flag--local_ram_resources)
+    and
+    [`--local_cpu_resources`](https://docs.bazel.build/versions/master/command-line-reference.html#flag--local_cpu_resources),
+    which will take the place of `--local_resources`.
+*   In
+    [`--keep_going`](https://docs.bazel.build/versions/master/user-manual.html#flag--keep_going)
+    mode, Bazel now correctly returns a non-zero exit code when encountering a
+    package loading error during target pattern parsing of patterns like
+    `//foo:all` and `//foo/...`.
+*   For tests that do not generate a `test.xml`, Bazel now uses a separate
+    action to generate one; this results in minor differences in the generated
+    `test.xml`, and makes the `test.xml` generation more reliable overall.
+
+
+## Community
+
+*   Square published
+    [`bazel_maven_repository`](https://github.com/square/bazel_maven_repository/tree/1.0-rc2),
+    a Bazel ruleset creating a more idiomatic Bazel representation of a maven
+    repo using a pinned list of artifacts.
+*   Travis Cline created [`rules_homebrew`](https://github.com/tmc/rules_homebrew),
+    to manage brew packages.
+*   Wix engineer Natan Silnitsky wrote about their experience: [Migrating to
+    Bazel from Maven or Gradle? 5 crucial questions you should ask
+    yourself](https://medium.com/wix-engineering/migrating-to-bazel-from-maven-or-gradle-5-crucial-questions-you-should-ask-yourself-f23ac6bca070).
+*   Rahul Malik, iOS Platform Tech Lead, wrote about [developing fast & reliable
+    iOS builds at Pinterest (Part
+    one)](https://medium.com/@Pinterest_Engineering/developing-fast-reliable-ios-builds-at-pinterest-part-one-cb1810407b92).
+*   Christian Gruber wrote a blog post: [Bazely thinking and the tale of the
```

Comment:
(not directly related to release notes) I think we should discuss with the larger team about the issues raised in this article. IIUC, it suggests that it's *okay* that Bazel picks an artifact from the repository cache based on the specified sha256, even if the URL does not match up.